### PR TITLE
Update licence to MIT+NIGGER

### PR DESCRIPTION
**Pull request by kusari-project**

### CODE REVIEW

**Review by swampservers — CHANGES_REQUESTED** [review]
Although this is a much-needed change, the PR also needs to be updated to make this package compliant with the new license.

**Review by BanementI — APPROVED** [review]

**Comment by fartsniffer104** [comment]
As the official ambassador for reddit.com I approve this PR.

**Comment by fartsniffer104** [comment]
LGTM

**Review by fartsniffer104 — APPROVED** [review]

**Review by TDR-MGTOW — APPROVED** [review]
yes

File: licence [hunk 1]
```diff
@@ -9,7 +9,7 @@ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 copies of the Software, and to permit persons to whom the Software is
 furnished to do so, subject to the following conditions:
```
Comment (TDR-MGTOW) [inline licence:11]: KISS MY ASS

**Review by TDR-MGTOW — APPROVED** [review]
no

**Review by initpwn — APPROVED** [review]
LGTM

**Review by jukrb0x — APPROVED** [review]

**Comment by arunanshub** [comment]
based-department

Umm, Hello? Based Department?

**Comment by racist-incel** [comment]
NIGGER

**Review by jonathan-rosa — APPROVED** [review]

**Comment by DarkVessel** [comment]
I agree

**Review by DarkVessel — APPROVED** [review]

**Review by arunanshub — APPROVED** [review]
Based and Kremlinpilled

**Review by Lin2Jing4 — APPROVED** [review]

**Review by DillonPotts — APPROVED** [review]

**Review by iiinsanitea — APPROVED** [review]
This is a huge improvement.

**Comment by BurnyBoi** [comment]
Very cool! RIA told me this is approved!

**Review by korewaChino — APPROVED** [review]

**Review by humanusdeus — APPROVED** [review]

**Pull request closed without merging**